web/src/components/ChecklistTag.tsx

```diff
@@ -0,0 +1,71 @@
+import { Icon } from "@/components/njwds/Icon";
+import { getMergedConfig } from "@/contexts/configContext";
+import { CheckoffStatus } from "@businessnjgovnavigator/shared/license";
+import { ReactElement } from "react";
+
+interface Props {
+  status: CheckoffStatus;
+}
+
+interface LicenseStatusMetadata {
+  iconName: string;
+  iconColor: string;
+  statusText: string;
+  statusTextColor: string;
+  backgroundColor: string;
+}
+
+const Config = getMergedConfig();
+
+export const statusObj: Record<CheckoffStatus, LicenseStatusMetadata> = {
+  ACTIVE: {
+    iconName: "check",
+    iconColor: "text-success",
+    statusText: Config.licenseSearchTask.completedStatusText,
+    statusTextColor: "text-base-darkest",
+    backgroundColor: "bg-base-lightest",
+  },
+  PENDING: {
+    iconName: "schedule",
+    iconColor: "text-base",
+    statusText: Config.licenseSearchTask.pendingPermitStatusText,
+    statusTextColor: "text-base-darkest",
+    backgroundColor: "bg-base-lightest",
+  },
+  SCHEDULED: {
+    iconName: "event",
+    iconColor: "text-base",
+    statusText: Config.licenseSearchTask.scheduledStatusText,
+    statusTextColor: "text-base-darkest",
+    backgroundColor: "bg-base-lightest",
+  },
+  INCOMPLETE: {
+    iconName: "close",
+    iconColor: "text-base",
+    statusText: Config.licenseSearchTask.incompleteStatusText,
+    statusTextColor: "text-base-darkest",
+    backgroundColor: "bg-base-lightest",
+  },
+  NOT_APPLICABLE: {
+    iconName: "do-not-disturb",
+    iconColor: "text-base",
+    statusText: Config.licenseSearchTask.notApplicableStatusText,
+    statusTextColor: "text-base-darkest",
```

Comment:
Since this class is the same across the board, you can remove it here and place it directly in line 68

Reply:
Good catch. Thanks!